packages/common/src/services/integrations/jwp/JWPAPIService.ts

```diff
@@ -0,0 +1,142 @@
+import { inject, injectable } from 'inversify';
+
+import StorageService from '../../StorageService';
+
+import type { InPlayerError } from './types';
+
+const INPLAYER_TOKEN_KEY = 'inplayer_token';
+const INPLAYER_IOT_KEY = 'inplayer_iot';
+
+const CONTENT_TYPES = {
+  json: 'application/json',
+  form: 'application/x-www-form-urlencoded',
+};
+
+type RequestOptions = {
+  withAuthentication?: boolean;
+  keepalive?: boolean;
+  contentType?: keyof typeof CONTENT_TYPES;
+  responseType?: 'json' | 'blob';
+  includeFullResponse?: boolean;
+};
+
+@injectable()
+export default class JWPAPIService {
+  private readonly storageService: StorageService;
+
+  private sandbox = true;
+
+  static isCommonError = (error: unknown): error is InPlayerError => {
```

Comment:
```suggestion
  static isCommonError = (error: unknown): error is JWPError => {
```

Reply:
Same for constants. I think we can use JWP now

Reply:
I just did that. This is the second time today this happening.